### ISSUE TITLE
Filters items based on their location

### ISSUE THREAD

**Issue by steenert** [arkayenro/arkinventory#1619]
**What version number you are using?**
v3.09.68

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
Retail

**What language is the game client set to?**
English

**Ask your question**
Is it possible to define a rule that would only display items when they are in your bank, not in your inventory, and not currently on the AH? I tried the following but it did not work:

loc( "bank" ) not loc(10) not loc("bag")

I want to post only single gear pieces on the AH and grab them from my bank, but only be visible if they are not in by bag or already on the AH.

Thanks in advance!

**Comment by arkayenro** [comment]
no, the rule functions only have access to the active slot and the item thats in it.  they cant tell if you have copies of the item anywhere else.

the `loc()` function was for limiting a rule to only applying in a certain location.
